src/odemis/driver/xt_client.py

```diff
@@ -1691,6 +1693,11 @@ def _setExternal(self, external):
         """
         scan_mode = "external" if external else "full_frame"
         self.parent.set_scan_mode(scan_mode)
+        # The dwellTime VA setter can only reflect changes on the SEM server side (parent)
+        # after the external VA is set to False i.e. 'full_frame'
+        if not external:
+            if self._current_dwell_time != self.parent.get_dwell_time():
+                self.dwellTime.value = self._current_dwell_time
```

Comment:
Is there the same issue with resolution? If so, it'd make sense to fix it at the same time.

Reply:
The resolution is a readonly VA [see here](https://github.com/delmic/odemis/blob/fb0fe2ad614bb03847647717748046dde4bdcbe1/src/odemis/driver/xt_client.py#L1485), which is used for calculation and setting  the scale VA (in the driver and [fastem_conf](https://github.com/delmic/odemis/blob/fb0fe2ad614bb03847647717748046dde4bdcbe1/src/odemis/acq/fastem_conf.py#L114)). I think there is no need to fix this, what do you think?

Reply:
Indeed... but then what about the scale?

Reply:
@tepals can the scale be set in the TFS UI when in `external` mode?

Reply:
Possible fixes:

```
        if not external:
            if self.dwellTime.value != self.parent.get_dwell_time():
                # Set the VA value again to reflect changes on the parent
                self.dwellTime.value = self.dwellTime.value
            if self.resolution.value != tuple(self.parent.get_resolution()):
                # Set the VA value again to reflect changes on the parent
                self.scale.value = self.scale.value
```

In below we don't call `self.parent.set_resolution(res)`
```
        if not external:
            if self.dwellTime.value != self.parent.get_dwell_time():
                # Set the VA value again to reflect changes on the parent
                self.dwellTime.value = self.dwellTime.value
            self._updateResolution()
```

Reply:
The scale is specific to Odemis, it updates the resolution and that is what we set on the TFS interface.

Reply:
Thanks Thera, fix 1. is needed then